XiEditor/EditViewController.swift

```diff
@@ -480,21 +505,30 @@ class EditViewController: NSViewController, EditViewDataSource, FindDelegate, Sc
         if !editView.isFirstResponder {
             editView.window?.makeFirstResponder(editView)
         }
+        infoPopover.performClose(self)
         editView.unmarkText()
         editView.inputContext?.discardMarkedText()
         let position  = editView.bufferPositionFromPoint(theEvent.locationInWindow)
         lastDragPosition = position
-        
-        document.sendRpcAsync("gesture", params: [
-            "line": position.line,
-            "col": position.column,
-            "ty": clickGestureType(event: theEvent)
-        ])
-        
+
+        let gestureType = clickGestureType(event: theEvent)
+
+        if gestureType == "request_hover" {
+            document.sendRpcAsync("request_hover", params: ["request_id": hoverRequestID, "position": ["type": "utf8_line_char", "line": position.line, "character": position.column]])
+            hoverEvent = theEvent
+            hoverRequestID += 1
```

Comment:
we don't seem to actually ever use this anywhere. Perhaps it's unnecessary?

Reply:
It is used for when an Option+Click is detected.